paddle/infrt/host_context/kernel_registry.h

```diff
@@ -23,7 +24,7 @@ namespace host_context {
 
 class KernelFrame;
 
-using KernelImplementation = void (*)(KernelFrame *frame);
+using KernelImplementation = std::function<void(KernelFrame *frame)>;
```

Comment:
```c++
auto kernel = [infershape_launcher](KernelFrame* frame) {
  return infershape_launcher(frame);
};

auto kernel = [](KernelFrame* frame) {
  CreateUnintedTensor(frame);
};
```

Hold the InferShapedKernelLauncher instance in the lambda function?

Reply:
I fixed it in commit [a759224](https://github.com/PaddlePaddle/Paddle/pull/39583/commits/a759224bdc11e1d1458e83b9c8610cdf385499ca)